e2e-tests/test/evm-apis.test.ts

```diff
@@ -130,3 +130,22 @@ describe("evm_revert", function () {
     expect(reverted).to.be.true;
   });
 });
+
+describe("evm_setBlockGasLimit", function () {
+  it("Should change block gas limit", async function () {
+    // Arrange
+    const wallet = new Wallet(RichAccounts[0].PrivateKey, provider);
+
+    // Act
+    const blockBefore = await provider.getBlock("latest");
+    const limitBefore = BigNumber.from(blockBefore.gasLimit).toString();
+    await provider.send("evm_setBlockGasLimit", ["0x1000000000000"]);
+    await provider.send("evm_mine");
+    const blockAfter = await provider.getBlock("latest");
+    await provider.send("evm_setBlockGasLimit", [limitBefore]);
```

Comment:
that looks useless

Reply:
I'd rather the test leave the limit at default - it might interfere with other tests...